root/static/scripts/tests/index-web.js

```diff
@@ -11,7 +11,7 @@ require('./GuessCase');
 require('./guessFeat');
 require('./i18n');
 require('./i18n/expand2');
-require('./relationship-editor');
+//require('./relationship-editor');
```

Comment:
This seems unrelated from the other changes? (it also causes an eslint warning)

Reply:
Oops, I had commented out that test locally because it was failing for some reason, then accidentally committed it.